main.go

```diff
@@ -194,7 +194,7 @@ func (s *Server) Serve() {
 }
 
 func (s *Server) getGlobalASN() (numorstring.ASNumber, error) {
```

Comment:
I think the name of this method is off now - perhaps getNodeASN(), or perhaps  getOurNodeASN() *and* rename getPeerASN to getNodeASN()

WDYT?